pkgs/development/python-modules/qpageview/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, python3
+, buildPythonPackage
+, python3Packages
+, pythonOlder
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

```suggestion
python3Packages.buildPythonPackage rec {
```
Also remove the now unused `python3` argument above.